File: from-1701-to-1800/1791-find-center-of-star-graph/solution.go
func findCenter(edges [][]int) int {
    e := map[int]bool{}
    for i := 0; i < 2; i ++ {
        v := edges[i][0]
        if e[v] {
            return v
        }
        e[v] = true
        v = edges[i][1]
        if e[v] {
            return v
        }
        e[v] = true
    }
    return -1
}

File: from-1701-to-1800/1791-find-center-of-star-graph/solution.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        vrt = dict()
        for v1, v2 in edges:
            if vrt.get(v1, False):
                return v1
            vrt[v1] = True
            if vrt.get(v2, False):
                return v2
            vrt[v2] = True

        return -1